### ISSUE TITLE
Right side of page background cut off. 

### ISSUE THREAD

**Issue by mcarreon** [darenju/react-flip-page#76]
The background color gets cut off on the right side of the page even when set to height 100%. 

Image of issue: [image](http://puu.sh/BRDaB/0ab36847be.png)

I'm got around it by using a css selector, but that has a delay in filling the component. 
.storybook is a classname of FlipPage
```css

.storybook > div > div > div > div> div {
  height: 100%;
}

```

Is this a possible fix? In react-flip-page/generateStyles.js the object on line 82

```js

pull: {
    left: orientation === 'vertical' ? 0 : '-100%',
    position: 'absolute',
    right: 0,
    height: 100%,
    top: orientation === 'vertical' ? '-100%' : 0,
  },

```

**Comment by darenju** [comment]
Have you tried your fix? Also, `100%` should be in quotes on the `height` property.

**Comment by mcarreon** [comment]
Hello! Yeah I tried out the fix, it seems to work and didn't seem to break
anything

**Comment by darenju** [comment]
Your fix should be implemented in `v1.2.2` @mcarreon. Care to check out?

**Comment by mcarreon** [comment]
Looks good!
